WcaOnRails/app/views/competitions/_competition_schedule_for_venue_table.html.erb

```diff
@@ -0,0 +1,83 @@
+<%# Local variables that must be passed: %>
+<%#   - competition %>
+<%#   - activities %>
+<%#   - rounds_by_wcif_id %>
+
+<%# The first 10 chars of iso8601 string are "YYYY-MM-DD" %>
+<% activities_by_day = activities.group_by { |a| a[:start].iso8601[0..9] } %>
```

Comment:
I'm pretty sure I did this because of some issues with timezones, but since the `a[:start]` is expressed in the local timezone, using `strftime` surely should work.